app/src/main/java/com/duycuong/weather/data/source/remote/WeatherRemoteDataSource.java

```diff
@@ -0,0 +1,127 @@
+package com.duycuong.weather.data.source.remote;
+
+import android.content.res.Resources;
+import android.location.Location;
+import android.util.Log;
+
+import com.duycuong.weather.BuildConfig;
+import com.duycuong.weather.data.model.AddressResponse;
+import com.duycuong.weather.data.model.CurrentlyResponse;
+import com.duycuong.weather.data.source.WeatherDataSource;
+import com.duycuong.weather.data.source.WeatherRepository;
```

Comment:
ERROR: Unused import: com.duycuong.weather.data.source.WeatherRepository. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports